curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/use-recursion-to-create-a-countdown.md

```diff
@@ -49,9 +49,10 @@ assert.isEmpty(countdown(-1));
 assert.deepStrictEqual(countdown(10), [10, 9, 8, 7, 6, 5, 4, 3, 2, 1]);
 ```
 
-`countdown(5)` should return `[5, 4, 3, 2, 1]`
+`countdown(5)` should return `[5, 4, 3, 2, 1]` and not abuse the global space
```

Comment:
I am not sure that this phrase is exactly what we want to use. I think that's more jargon-y than we want? 

 Perhaps 'Global variables should not be used to cache the array.' should be in the instructions and then used as a separate test text?

Reply:
Yes, ive completed that.